File: 151001/Shlyk/SpringCalculator/spring-calculator-rest/src/main/java/by/bsuir/poit/dc/rest/api/dto/response/NewsDto.java
package by.bsuir.poit.dc.rest.api.dto.response;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.baidu.bjf.remoting.protobuf.annotation.ProtobufClass;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.redis.core.RedisHash;

import java.io.Serializable;
import java.sql.Timestamp;
import java.util.Date;

/**
 * DTO for {@link by.bsuir.poit.dc.rest.model.News}
 */
@Data
@Builder
@ProtobufClass
@AllArgsConstructor
@NoArgsConstructor
public class NewsDto {
    private long id;
    private String title;
    private String content;
    private Date created;
    private Date modified;
    private long userId;
}

File: 151004/Pyshny/RV/RV/Views/DTO/NewsDTO.cs
using System.ComponentModel.DataAnnotations;

namespace RV.Views.DTO
{
    public class NewsDTO
    {
        [Required]
        public int id { get; set; }
        [Required]
        public int userId { get; set; }
        [Required]
        public string title { get; set; }
        [Required]
        public string content { get; set; }
        [Required]
        public DateTime created { get; set; }
        [Required]
        public DateTime modified { get; set; }
    }
}
